guides/common/modules/proc_rex-pull-based-performance-impact-considerations.adoc

```diff
@@ -0,0 +1,24 @@
+[id="REX_pull_based_performance_impact_considerations_{context}"]
+= REX pull-based transport performance impact considerations
+
+When {ProjectServer} is configured to use REX pull-based transport for remote execution jobs using "Script" provider, {SmartProxyServer} sends a notification about new job to the client machines via MQTT. This notification does not include actual workload (actual script) the client is supposed to execute. Once a client receives a message about a new remote execution job, it queries the {SmartProxyServer} for actual payload. As the client executes the job, it periodically sends outputs of the job to the capsule, further increasing the number of requests to the {SmartProxyServer}.
```

Comment:
Please stick to our convention "one sentence per line".

Reply:
my bad, I forgot. Updated.